src/module.c

```diff
@@ -943,6 +943,20 @@ ValkeyModuleCtx *moduleAllocateContext(void) {
     return (ValkeyModuleCtx *)zcalloc(sizeof(ValkeyModuleCtx));
 }
 
+static void computeNextYieldTime(ValkeyModuleCtx *ctx) {
+    /* in loading we depend on the server hz, but in other cases we also wait
```

Comment:
Looks like you dropped a line from the comment.

Reply:
it is intentional. the line I dropped is about the caller.

```c
    /* Calculate the initial yield time for long blocked contexts. */
```